docs/user-guide/administration/design-table.md

```diff
@@ -19,220 +19,257 @@ Before proceeding, please review the GreptimeDB [Data Model Documentation](/user
-  PRIMARY KEY(host, idc),
-  TIME INDEX(ts)
-);
+SELECT message FROM http_logs WHERE application = 'greptimedb' AND access_time > now() - '5 minute'::INTERVAL;
 ```
 
-Here, `host` and `idc` are primary key columns and Tag columns, and `ts` is the `TIME INDEX`. Other fields like `cpu_util` are Field columns.
+The number of applications is usually limited. Table `http_logs_v2` uses `application` as a tag.
+It sorts logs by application so querying logs under the same application is faster.
 
-However, this design has limitations. Specifically, it does not support deduplication and optimized sorting for certain columns without creating additional indexes, which can lead to data expansion and performance degradation.
+```sql
+CREATE TABLE http_logs_v2 (
+  access_time TIMESTAMP TIME INDEX,
+  application STRING,
+  remote_addr STRING,
+  http_status STRING,
+  http_method STRING,
+  http_refer STRING,
+  user_agent STRING,
+  request_id STRING,
+  request STRING,
+  PRIMARY KEY(application),
+) with ('append_mode'='true');
+```
+
+The unique values of the primary key in the table can significantly affect performance.
+Currently, the recommended number of values for the primary key is no more than 100 thousand.
+So you must not use high cardinality column as the primary key or put too many columns in the primary key.
 
-For instance, in a monitoring scenario involving serverless containers,
-the host names of these short-lived containers can cause high cardinality issues if added to the primary key.
-Despite this, deduplication based on host names is still desired.
-Similarly, in IoT scenarios, there may be tens of thousands of devices.
-Adding their IP addresses to the primary key can also result in high cardinality problems,
-yet deduplication based on IP addresses remains necessary.
+Besides primary key, you can also use index to accelerate specific queries on demand.
 
-## Separating Primary Key and Inverted Index
+### Inverted Index
 
-Therefore, starting from `v0.10`, GreptimeDB supports separating the primary key and index.
+GreptimeDB supports inverted index that may speed up filtering low cardinality columns.
 When creating a table, you can specify the [inverted index](/contributor-guide/datanode/data-persistence-indexing.md#inverted-index) columns using the `INVERTED INDEX` clause.
-In this case, the `PRIMARY KEY` will not be automatically indexed but will only be used for deduplication and sorting:
+For example, `http_logs_v2` adds an inverted index for the `http_method` column.
 
+```sql
+CREATE TABLE http_logs_v2 (
+  access_time TIMESTAMP TIME INDEX,
+  application STRING,
+  remote_addr STRING,
+  http_status STRING,
+  http_method STRING INVERTED INDEX,
+  http_refer STRING,
+  user_agent STRING,
+  request_id STRING,
+  request STRING,
+  PRIMARY KEY(application),
+) with ('append_mode'='true');
+```
 
-Example:
+The following query can use the inverted index on the `http_method` column.
 
 ```sql
```

Comment:
Do we need to explain which type of queries can benefit from indexes? Only the exact equal or prefix matching or range queries like that.